### PR TITLE
Defining some verbs to use

### PR DESCRIPTION
**Pull request by mosch**
Proposal. I was running into checking how other methods are named – I think this is a good place to document it.

### CODE REVIEW

File: design.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
 
 ## Rest SDK
 
+### Action Verbs
+
+✅get <br/>
```
Comment (iamn1ck) [inline design.md]: We are using Find for some methods currently, maybe you could go rename those functions with the verb get

**Comment by mosch** [comment]
Did. I guess it's a breaking one.

File: design.md (hunk 1 above)
Comment (adieuadieu) [reply design.md]: @iamn1ck yea—we should rename that method. There are still a couple of methods which don't adhere to our design guide. E.g. `getUsers()`

https://github.com/allthings/node-sdk/blob/f761106dcfd921d929e3815e8fa4d1ca3a6d2cd6/src/rest/methods/user.ts#L112-L124

**Comment by adieuadieu** [comment]
@mosch @iamn1ck — I think this is ready to be merged. Shall we?

**Pull request merged**